shared/dataflow/codeql/dataflow/DataFlow.qll

```diff
@@ -150,6 +176,9 @@ signature module InputSig {
     Content getAReadContent();
   }
 
+  /**
+   * A content approximation.
+   */
```

Comment:
What is a content approximation?

Reply:
My guess: it covers a lot of different contents, but still provides some value in distinguishing some different contents.

Reply:
Yes. It's used to provide an in-between level of precision to make more gradual pruning possible. We have plans to internalise this concept into the library, such that it won't be necessary to expose in the interface like this. But it may take a while before we get to that.

Reply:
I'll try and come up with some words for it...

Reply:
Done.  I've also documented `getContentApprox` below this class.  I haven't included a lot of detail, because I don't think this is one of the more essential features in this file.